### PR TITLE
Ddos/release 2022 04 07

### PR DESCRIPTION
**Pull request by jdesgats**
Note: the two `scheduled.json` files will just contain scheduled changes in a JSON format to help the changelog generator from one release to the next (in order to filter previously scheduled changes correctly when re-generating the document).

### CODE REVIEW

File: content/ddos-protection/change-log/http/2022-04-07.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <th>Rule ID</th>
+      <th>Description</th>
+      <th>Previous Action</th>
+      <th>New Action</th>
+      <th>Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>c9f18c647ae745c6b81b459d8ed59b32</td>
```
Comment (pedrosousa) [inline content/ddos-protection/change-log/http/2022-04-07.md]: Rule IDs are usually trimmed in the WAF change log, keeping the last 8 characters of the ID (that's what users see in the dashboard rules list). Would it make sense to do the same here...?

File: content/ddos-protection/change-log/http/2022-04-07.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: 2022-04-07
+type: table
```
Comment (pedrosousa) [inline content/ddos-protection/change-log/http/2022-04-07.md:3]: We don't really need this line with the new docs tooling (we will eventually remove these from the WAF change log).

**Review by pedrosousa — APPROVED** [review]
Added a few comments that can be addressed later. Approving because the new change log page mentions today's date.

File: content/ddos-protection/change-log/http/2022-04-07.md (hunk 1 above)
Comment (jdesgats) [reply content/ddos-protection/change-log/http/2022-04-07.md]: Done, I'll update the generator to do this for the next release.

File: content/ddos-protection/change-log/http/2022-04-07.md (hunk 2 above)
Comment (jdesgats) [reply content/ddos-protection/change-log/http/2022-04-07.md:3]: Okay, I just copy/pasted the WAF skeleton, I'll remove it from my template for next releases.

**Pull request merged**